src/scanpy/_utils/__init__.py

```diff
@@ -247,14 +247,18 @@ def _check_array_function_arguments(**kwargs):
         )
 
 
-def _check_use_raw(adata: AnnData, use_raw: None | bool) -> bool:
+def _check_use_raw(
+    adata: AnnData, use_raw: None | bool, layer: str | None = None
```

Comment:
```suggestion
    adata: AnnData, *, use_raw: None | bool, layer: str | None = None
```

since it’s no longer only one parameter aside from adata, better specify them by name

Reply:
This would break a lot of functions if i would change this

Reply:
Then `layer=layer` should probably be added in most of these other places as well, so this PR should touch these anyway, no?

Reply:
I think we should leave the plotting alone for the time beeing